### DISCUSSION TITLE
[RFC] UI Frameworks

### DISCUSSION THREAD

**Discussion by dcyoung-dev** [DCC-EX/EX-WebThrottle2#30] [Ideas]
### Problem

We currently use [Vuetify](https://vuetifyjs.com/en/) to provide some UI components and styling.

Our designs in Figma have a different feel to them compared to the [material style](https://m2.material.io/design) of Vuetify.

### Reasoning

As we develop the application, making it look and feel like our designs, we will come up against the design limitations of the Vuetify framework, which will make development more difficult. i.e. we'll spend effort making Vuetify look like our designs.

A bonus to frameworks like Vuetify is that they provide useful interactive components, i.e. the [Tabs](https://vuetifyjs.com/en/components/tabs/) component, which make the development process easier.

### Questions

1. Should we continue with Vuetify?
2. What alternatives could we use?
3. Should we change our designs to match the flat style of Vuetify?

**Comment by dexslab** [comment]
Vuetify is just a framework for components. It has full customization ability like most other frameworks. I added it because i am used to using and i could get it going quickly i have some code i need to push. As for everything else we could use tailwind but again all the CSS is going to come down to @ManiAkasapu since he is the UI master and designed what we have so far. Whatever we use will have to be customized to our look at feel. Think of the UI frame work as a blank lego bricks. The y provide us the tools needed to build an awesome UI but it still needs to be painted to match what we are looking for which is where Mani comes in.
